JSDemos/Demos/Popup/Overview/Angular/description.md

```diff
@@ -1,10 +1,34 @@
-This demo illustrates the Popup component. Click the Details button under an employee picture to see what the Popup looks like. Here is how you can configure its main elements:
+This demo shows how to show and hide the Popup component, how to fill it with content, specify its position and other configurations. To get started with the DevExtreme Popup component, refer to the following tutorial for step-by-step instructions: [Getting Started with Popup](/Documentation/Guide/UI_Components/Popup/Getting_Started_with_Popup/).
 
-- **Content**        
-Put any valid markup inside the component. 
+## Show and Hide the Popup
 
-- **Title Bar**       
-Set [showTitle](Documentation/ApiReference/UI_Components/dxPopup/Configuration/#showTitle) to **true** and use the [title](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#title) property to specify the caption. 
+Use two-way binding to define the [visible](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#visible) property and bind it to a boolean variable, `popupVisible` in this demo. The `showInfo(employee)` method assigns `true` to the `popupVisible` variable, and the Popup becomes visible. You can close the Popup in multiple ways:
```

Comment:
- "use two-way binding ... and bind" - similar words in close succession. 
- "boolean" - should be capitalized, because it comes from a last name. 
- "becomes visible" - "appears" is shorter. Be on the lookout for phrases that combine a common verb plus an adjective. Usually a single verb exist that can be used as a replacement. For example "get bigger" = "grow", "make simpler" = "simplify".

```suggestion
Use two-way binding between the [visible](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#visible) property and a Boolean variable (`popupVisible` in this demo). The `showInfo(employee)` method sets `popupVisible` to `true` and the Popup appears. You can close the Popup in multiple ways:
```